bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -14,8 +14,11 @@
 {% block smarton_content %}
 <article class="main">
 
+  {# see bug 1188591, comment 10 for details regarding the need for the en-* conditional #}
+  {% set tweet_text = _('Stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') if LANG.startswith('en-') else _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') %}
```

Comment:
Nit: would this be a little more readable on more than one line?

    {% if LANG.startswith('en-') %}
      {% set tweet_text = _('Stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') %}
    {% else %}
      {% set tweet_text = _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') %}
    {% endif %}